### ISSUE TITLE
1. Should have aria-haspopup and aria-expanded labels

### ISSUE THREAD

**Issue by zcjess** [zcjess/GradeCraft#19]
1. Should have aria-haspopup and aria-expanded labels
2. Should be able to close the popup by clicking the button again, right now no way to close it
<img width="1274" alt="image" src="https://github.com/zcjess/GradeCraft/assets/90011962/50b70e51-9ab6-47f5-81f0-944369f29568">

_Originally posted by @zcjess in https://github.com/zcjess/GradeCraft/issues/1#issuecomment-1660979956_
